cinema/views.py

```diff
@@ -34,6 +35,16 @@ class MovieViewSet(viewsets.ModelViewSet):
     queryset = Movie.objects.all()
     serializer_class = MovieSerializer
 
+    @staticmethod
+    def _params_to_inits(query_string):
+        """
+        Converts a params string of format 1, 2, ... n,
+        to list format [1, 2, ..., n]
+        """
+        return [
+            int(str_id) for str_id in query_string.split(",")
```

Comment:
Consider adding error handling for the conversion of `str_id` to `int` in `_params_to_inits`. If the query string contains non-integer values, it will raise a ValueError.